UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -3593,4 +3616,24 @@ public DescriptionView(string heading, string description)
             Description = description;
         }
     }
+
+    public class StringTitleConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is not string str)
+                return null;
+
+            if (str.Length > 256)
+                str = str[..256] + "...";
+            str = str.Replace('\n', ' ');
+
+            return str;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
+        }
+    }
```

Comment:
I thought that it's used only by `MainWindow` (and can't be useful elsewhere), so it should be there

Reply:
MainWindow currently is at ~5k lines if memory serves me correctly, I'd rather have it separate file. If you want to, you can still put it inside MainWindow, since its a partial class.

Reply:
Oh, then yeah